packages/conferer/src/Conferer/FromConfig/Internal.hs

```diff
@@ -367,14 +371,14 @@ instance Show TypeMismatchWithDefault where
     ]
 
 -- | Helper function to throw a 'TypeMismatchWithDefault'
-throwTypeMismatchWithDefault :: forall a b. (Typeable a) => Key -> Dynamic -> IO b
-throwTypeMismatchWithDefault key dynamic =
-  throwIO $ typeMismatchWithDefault @a  key dynamic
+throwTypeMismatchWithDefault :: forall a b. (Typeable a) => Key -> [Dynamic] -> IO b
```

Comment:
Are these functions still being used?, here it was replaced by another function:
https://github.com/ludat/conferer/pull/47/files#diff-35cf0debc54ed900e19983b1cab57574222bd34bd8132285ab64b82b4225d0bcR254